### PR TITLE
Location page

### PR DESCRIPTION
**Pull request by dreamingOfSoy**

### CODE REVIEW

File: wild-swimming-app/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@auth0/auth0-react": "^2.0.2",
-    "axios": "^1.4.0"
+    "axios": "^1.4.0",
```
Comment (amnioc) [inline wild-swimming-app/package.json:14]: also did this on mine, great spot

File: wild-swimming-app/src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-      <Route path="terms" element={<TsCs />} />
-    </Route>
-  )
-);
 
 function App() {
-  return <RouterProvider router={router} />;
+  return (
+    <section>
+      <Routes>
```
Comment (amnioc) [inline wild-swimming-app/src/App.tsx:12]: this layout is nice and clean! thank you!

File: wild-swimming-app/src/pages/location-details/Location-details.tsx [hunk 3]
```diff
@@ -1,5 +1,61 @@
...
+    };
+
+    sendLocationRequest(
+      "GET",
+      `https://environment.data.gov.uk/data/bathing-water-profile/${id}/2023:1`,
+      res
+    );
+  }, [sendLocationRequest, id]);
+
+  console.log(location);
```
Comment (amnioc) [inline wild-swimming-app/src/pages/location-details/Location-details.tsx:31]: keeping for later? just making comment so we remember to take it out.

**Review by amnioc — APPROVED** [review]
this looks great, cool creations and big thank you :)

File: wild-swimming-app/src/pages/location-details/Location-details.tsx (hunk 3 above)
Comment (dreamingOfSoy) [reply wild-swimming-app/src/pages/location-details/Location-details.tsx:31]: aye, just wanted to push this so people can use the new router code quicker, will remove once I've done page this aft. Thanks, Nico!

**Pull request merged**